app/src/main/scala/org/alephium/explorer/persistence/DBRunner.scala

```diff
@@ -32,3 +32,13 @@ trait DBRunner {
       case error => throw new RuntimeException(error)
     }
 }
+
+object DBRunner {
+  @SuppressWarnings(Array("org.wartremover.warts.Throw"))
+  def run[R, E <: Effect](databaseConfig: DatabaseConfig[PostgresProfile])(action: DBAction[R, E])(
+      implicit executionContext: ExecutionContext): Future[R] =
+    databaseConfig.db.run(action).recover {
+      case error => throw new RuntimeException(error)
```

Comment:
Is it common practice to throw RuntimeException for any DB error? cc @simerplaha

Reply:
Think it is good idea to wrap it in some `OurDBException` so it is easier to trace DB issues in the logs or in extreme cases implement some recovery logic. I think that must have been the goal here.

Reply:
Not to blame anything, just to raise the topic of error handling here. It's a non-trivial topic

Reply:
We will need a task for DB error handling

Reply:
issue here: https://github.com/alephium/explorer-backend/issues/207